src/pymovements/dataset/dataset.py

```diff
@@ -234,6 +234,32 @@ def load_event_files(
         )
         return self
 
+    def apply(
+            self,
+            function: str,
+            *,
+            verbose: bool = True,
+            **kwargs: Any,
+    ) -> Dataset:
+        """Apply preprocessing method to all GazeDataFrames in Dataset.
+
+        Parameters
+        ----------
+        function: str
+            Name of the preprocessing function to apply.
+        verbose : bool
+            If True, show progress bar of computation.
+        kwargs:
+            kwargs that will be forwarded when calling the preprocessing method.
```

Comment:
please add an example, because it is not straight forward for non experts

Reply:
I have added it as an example and to the tutorial notebook